assets/scripts/script.js

```diff
@@ -1,27 +1,32 @@
-// const check = document.getElementById("check");
-// const displayy = document.getElementById("display");
+const check = document.getElementById('check')
+const displayy = document.getElementById('display')
 
-// function palindrom() {
-//   let enterr = document.getElementById("start").value;
+function palindrom () {
+  const enterr = document.getElementById('start').value
+  let result = ''
+  if (enterr === '') {
+    displayy.innerHTML = ' type a text or number'
+  } else {
+    const restrictSymbols = enterr.replace(/[^a-z0-9]/g, '').toLowerCase()
+    result = restrictSymbols.split('').reverse().join('')
 
-//   let result = "";
-//   result = enterr.split("").reverse().join("");
+    // console.log('this is result', result)
```

Comment:
remove dead code in your final submission

Reply:
okay sir, thank you